src/sidebar/components/menu.js

```diff
@@ -9,6 +9,14 @@ const { listen } = require('../util/dom');
 
 const SvgIcon = require('./svg-icon');
 
+// The triangular indicator below the menu toggle button that visually links it
+// to the menu content.
+const menuArrow = (
+  <svg className="menu__arrow" width={15} height={8}>
```

Comment:
Are the braces on width and height because there are no quotes allowed on these attrs in SVG?